.circleci/config.yml

```diff
@@ -63,19 +62,31 @@ jobs:
     parallelism: 6
     steps:
       - checkout
-      - yarn_install
+      - node/install-packages:
+          pkg-manager: yarn-berry
       - run:
           name: Run Jest
           command: |
             yarn jest --ci --forceExit --logHeapUsage --runInBand --reporters=default --reporters=jest-junit --shard=$(expr $CIRCLE_NODE_INDEX + 1)/$CIRCLE_NODE_TOTAL
+  type-check:
+    executor:
+      name: node/default
+      tag: "22.5"
+    steps:
+      - checkout
+      - fix_corepack_permissions
```

Comment:
i wonder if this is the source of some of the permissions errors we have been running into in eigen 👀